requirements/testing.txt

```diff
@@ -136,7 +136,6 @@ websocket-client==1.2.0
     # via docker
 wrapt==1.12.1
     # via astroid
-
```

Comment:
Let's add the removed newline to avoid introducing unnecessary changes

Reply:
Removed in [78f625a](https://github.com/apache/superset/pull/21322/commits/78f625ac12c55f03f70fa88a01c3176e9e8de636)